### ISSUE TITLE
Website is not accessible for colourblind users (WCAG requirements)

### ISSUE THREAD

**Issue by sKhcodes** [ShrutiMishra-2002/hacktoberfest22_transopedia#2]
In order to make the website accessible to all trans people, including those with visual difficulties, you need to make sure that the text meets WCAG contrast requirements. 
A lot of people can't read blue text on a blue background for example.

If this looks like a good issue to you, I will go through and check all of your text and backgrounds against AA contrast requirements (or I can do AAA if you prefer) and then I will lighten or darken the text to make it readable for visually impaired users.
